aspnetcore/release-notes/aspnetcore-7.0.md

```diff
@@ -0,0 +1,45 @@
+---
+title: What's new in ASP.NET Core 7.0
+author: rick-anderson
+description: Learn about the new features in ASP.NET Core 7.0.
+ms.author: riande
+ms.custom: mvc
+ms.date: 10/29/2021
+uid: aspnetcore-7
+---
+# What's new in ASP.NET Core 7.0 preview
+
+This article highlights the most significant changes in ASP.NET Core 7.0 with links to relevant documentation.
+
+## API controllers
```

Comment:
This is just an initial document, right? I am asking because there a lot of features that need to be added.

Reply:
> This is just an initial document, right? I am asking because there a lot of features that need to be added.

Yes, I'll be updating this with each new preview release